docs/public/setup.md

```diff
@@ -33,10 +33,41 @@ On Windows, install Python 2.7 and/or 3.11 via the [official installer](https://
 
 ##### Preface
 
+[Invoke](http://www.pyinvoke.org) is a task runner written in Python that is extensively used in this project to orchestrate builds and test runs. To run the tasks, you need to have it installed on your machine. We offer two different ways to run our invoke tasks.
+
+##### `deva` (recommended)
```

Comment:
Could we factorise with what is on docs/dev/agent_dev_env.md?

Reply:
I'm afraid we can't right now sadly